recipes/openbabel/2.4.1/meta.yaml

```diff
@@ -35,12 +37,15 @@ requirements:
         - zlib
         - libxml2
         - cairo   # [linux]
+        - xorg-libxau
```

Comment:
Why do you need any xorg packages for OB? Are you trying to build OB-GUI? I don't think conda is a good place for it.